Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -1,5 +1,24 @@
 // ArrayExtensions.swift - Copyright 2020 SwifterSwift
 
+// MARK: - Initializers
+
+public extension Array {
+    /// SwifterSwift: Creates an array with specified number of elements, for each element it calls specified closure.
+    /// - Parameters:
+    ///   - numberOfElements: The number of elements in the new array.
+    ///   - element: A closure that initializes each element.
+    ///     - Parameter *index*: An index of initialized element in the array.
+    ///     - Returns: element of the array.
+    init(numberOfElements: Int, element: (Int) -> Element) {
```

Comment:
I guess this would be the same as 
https://github.com/SwifterSwift/SwifterSwift/blob/8ff36b66879c8d5eadfd54d2e33dafd62df6612b/Sources/SwifterSwift/SwiftStdlib/RangeReplaceableCollectionExtensions.swift#L15

Reply:
Except that the closure receives the index, which could be handy.
And the closure is a the end, which means that you can add an unnamed closure instead of having to name it.

Reply:
Ah makes sense!

Reply:
I would only give a minor suggestion then, is to use `count` for parameter name instead of `numberOfElements` because that is standard across apple APIs. But I'm fine with either ...

Reply:
I've renamed `numberOfElements` to `count`